### PR TITLE
Send logs as JSON

### PR DESCRIPTION
**Pull request by samcrang**
https://github.com/openregister/openregister-java/pull/276 needs to get merged first.

### CODE REVIEW

File: ansible/roles/openregister_config/templates/openregister-config.yaml.j2 [hunk 1]
```diff
@@ -58,6 +58,10 @@ downloadConfigs: true
 
 server:
   registerDefaultExceptionMappers: false
+  requestLog:
+    type: classic
```
Comment (endamccormack) [inline ansible/roles/openregister_config/templates/openregister-config.yaml.j2]: I cant see anything about this type attribute in the documentation. Whats it for?

Comment (samcrang) [inline ansible/roles/openregister_config/templates/openregister-config.yaml.j2]: hmmm, I _think_ it's related to the format that logback uses but I'll double check and see what it's actually doing.

Comment (samcrang) [inline ansible/roles/openregister_config/templates/openregister-config.yaml.j2]: I don't entirely understand why it's needed but without it it fails to log requests /shrug

Comment (samcrang) [inline ansible/roles/openregister_config/templates/openregister-config.yaml.j2]: @endamccormack https://github.com/dropwizard/dropwizard/blob/master/docs/source/manual/configuration.rst#classic-request-log

**Pull request merged**